src/homework1/AccountService.java

```diff
@@ -0,0 +1,76 @@
+import java.time.Month;
+import java.time.Year;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class AccountService {
+
+    public List<Account> over500Balance (List<Account> accounts){
+        return accounts.stream()
+                .filter(account -> account.getBalance() >= 500)
+                .toList();
+    }
+    public Set<String> listCountries(List<Account> accounts) {
+        return accounts.stream()
+                .map(Account::getCountry)
+                .collect(Collectors.toSet());
+    }
+
+    public boolean youngerThan2000Year(List<Account> accounts){
+        return accounts.stream()
+                .allMatch(account -> account.getBirthday().getYear() < 2000);
+    }
+
+    public double sumAllMaleBalance(List<Account> accounts){
+        return accounts.stream()
+                .filter(account -> account.getGender().equals("Male"))
```

Comment:
Add constant variable for string "Male" or use ENUM Gender

Reply:
then we should add constant variable for other methods too! i was thinking about doing this but i followed your tasks